### ISSUE TITLE
Push changes upstream

### ISSUE THREAD

**Issue by kingosticks** [southafricanrob/musicbox-webclient#1]
Hi,
Would you be interested in creating some PRs and getting your changes merged into https://github.com/woutervanwijk/Mopidy-MusicBox-Webclient ?

We've got open bugs for some of the fixes you've got, and most of the others sound like great changes. 

I'm also interested in your idea of displaying links to other local musicbox instances in the interface somewhere. We can probably automatically discover them with some python running in the server.

**Comment by southafricanrob** [comment]
Hi, Sure  - keen to help where i can. I don't really know too much about github at this stage - am i correct i should try create a fork of wouters webclient and then make the changes to that rather?

**Comment by kingosticks** [comment]
Yes please. You don't really have to bother with branches or anything if you don't want to.  But if you could split the different improvements up into different commits that would be really helpful when we come to review and merge it (but it's not the end of the world otherwise). Thanks very much.

**Comment by southafricanrob** [comment]
Cool - will play around tomorrow evening - it has been a while since i did any changes to the webclient so will have to get my head around it again!

**Comment by southafricanrob** [comment]
Hi I have mad some put some of the changes in a fork of the latest webclient and done 2 commits - will add more as i go along - hope it helps

**Comment by kingosticks** [comment]
Awesome. We are going to do a release in the next week or so, will be great
to include whatever is available.

**Comment by southafricanrob** [comment]
Hi Nick - do you need to me to make a pull request? STill understanding the basics of github forks....apologies!

Date: Tue, 10 Mar 2015 16:35:12 -0700

**Comment by kingosticks** [comment]
Yes please.
